src/views/catalog/wizard/tabs/network/WizardNetworkTab.tsx

```diff
@@ -3,24 +3,23 @@ import * as React from 'react';
 import { WizardTab } from '@catalog/wizard/tabs';
 import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
-import {
-  ListPageBody,
-  ListPageCreateButton,
-  ListPageHeader,
-} from '@openshift-console/dynamic-plugin-sdk';
+import { ListPageBody, ListPageCreateButton } from '@openshift-console/dynamic-plugin-sdk';
 
 import NetworkInterfaceList from './components/list/NetworkInterfaceList';
 import NetworkInterfaceModal from './components/modal/NetworkInterfaceModal';
 
+import 'src/utils/styles/ListPageCreateButton.scss';
```

Comment:
we should import absolute css files like this, only relatively.
please copy the css to a new file in this directory and `import './file.css`

Reply:
Is it optimal to copy the same file many times?